### PR TITLE
protonmail-bridge: update to 3.8.2.

### PR DESCRIPTION
**Pull request by jbenden**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, (`x86_64`-`glibc`)

### CODE REVIEW

File: srcpkgs/protonmail-bridge/template [hunk 1]
```diff
@@ -1,28 +1,25 @@
...
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="Joseph Benden <joe@benden.us>"
 license="GPL-3.0-or-later"
 homepage="https://proton.me/mail/bridge"
 distfiles="https://proton.me/download/bridge/protonmail-bridge_${version}-1_amd64.deb"
-checksum=4541f8aa145097ee171889b0c9d058450d09dc23be10231c83d6659789a27627
+checksum=410258231ac5df2f4896e9aa1c39cf6b7449fef79541ed31706ad4d8e134b545
 
 restricted=yes
+noverifyrdeps=yes
```
Comment (oreo639) [inline srcpkgs/protonmail-bridge/template:16]: Don't add `noverifyrdeps=yes` here, this causes *all* rundeps to not be discovered.
The missing rdeps are for qt6-webview, which isn't packaged currently, the fix would be to package the dependency.

Comment (jbenden) [reply srcpkgs/protonmail-bridge/template:16]: This is propriety software which bundles all dependencies. We cannot substitute our own.

Comment (oreo639) [reply srcpkgs/protonmail-bridge/template:16]: Ah, yeah, qt6-webview plugin is bundled. However `noverifyrdeps=yes` also causes dependencies like glibc, libxcb, glib, etc to not get pulled in, which are not bundled.

Comment (classabbyamp) [reply srcpkgs/protonmail-bridge/template:16]: > This is propriety software which bundles all dependencies. We cannot substitute our own.

isn't the source here? 
https://github.com/ProtonMail/proton-bridge

**Pull request closed without merging**